docs/docs/dev_docs/contracts/deploying.md

```diff
@@ -1,6 +1,6 @@
 # Deploying contracts
 
-Once you have [compiled](./compiling.md) your contracts you can proceed to deploying them using `aztec-cli`.
+Once you have [compiled](./compiling.md) your contracts you can proceed to deploying them using the aztec-cli or using aztec.js which is a Typescript client to interact with the sandbox.
 
 ## Prerequisites
 - aztec-cli installed (go to [CLI main section](./main.md) for installation instructions)
```

Comment:
This seems to be the wrong link? It is just pointing to contract, and not to the cli docs.